spec/views/welcome/index.html.erb_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+RSpec.describe "welcome/index.html.erb", type: :view do
+  it "renders a welcome message" do
+    render
+    expect(rendered).to have_selector("h1", text: "Notation")
```

Comment:
suggestion: Let's also expect a sign up/sign in button to be rendered and pointing to the correct url.

Reply:
ok